src/content/docs/en/guides/imports.mdx

```diff
@@ -200,6 +200,11 @@ Additionally, glob patterns must begin with one of the following:
  
 
 [Read more about the glob pattern syntax](https://github.com/mrmlnc/fast-glob#pattern-syntax).
+
+#### `Astro.glob()` vs Content Collections
+
+[Content collections](/en/guides/content-collections/) were introduced in `astro@2.0.0`. It's the best way to work with Markdown and MDX files in an Astro project. If your directory contains only Markdown and MDX files, it's best to use Content collections and the `src/content` directory.
```

Comment:
Thanks for this @donovangomez!

I’ll let @sarah11918 decide here, but I think we don’t generally tell people what is “best” but try to describe some differences and allow people to decide for themselves. Hard to do in a short paragraph where we don’t want to repeat to much of what’s in the content collections guide itself. Maybe something like this?

```suggestion
[Content collections](/en/guides/content-collections/) provide a `getCollection()` API for loading multiple files instead of `Astro.glob()`. If you are working with Markdown and MDX files, you can store them in the `src/content/` directory to take advantage of content collection features.
```

Reply:
Thanks @delucis ! My bad that was totally mentioned in the writing guide. I do like the rewrite a lot more, it's more neutral.

Reply:
Yup, @delucis' edit is much more in keeping with our docs style. I'm going to adjust it just a bit, to help make it more future-proof (e.g. Markdoc...) but it's absolutely the way to go!  :raised_hands: